src/Java_training3_1.java

```diff
@@ -0,0 +1,34 @@
+
+public class Java_training3_1 {
+    public static void main(String[] args){
+
+        //Task 1
+        int [] num = new int[2];
+        num[0] = 5;
+        num[1] = 15;
+
+
+        //Task 2
+        int [] numbers = new int [1000];
+        int number = 0;
+        while (number < numbers.length){
+            numbers[number] = number + 1;
+            number = number + 1;
+        }
+        System.out.println(numbers[500]);
```

Comment:
what do you think, wouldn't be better for this kind of task to use "for" instead of "while"?

Reply:
I think better to use "for" loop in this kind of tasks, but almost in tasks I have used "for" loop, for this in this task  I have used 'while' loop